recipes/prefsync/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.2" %}
+{% set sha256 = "66f0d3870b442e715de23b632e83f27f5971885b40bb3f6b211437138c554496" %}
+
+package:
+  name: prefsync
+  version: {{ version }}
+
+source:
+  fn: prefsync-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/prefsync/prefsync-{{ version }}.tar.gz
```

Comment:
Yeah, I made a similar error when I first saw the new URLs. The `s` based example might be an unfortunate choice as it is not too unreasonable to assume it stands for `source` or similar. I've been making my PyPI recipes like this [one]( https://github.com/conda-forge/funcsigs-feedstock/blob/f896b21228dd6dcaa74c2c60cad9b3f1b4648ebb/recipe/meta.yaml ) to keep life simple. Maybe we should change the example to match.